<launch>

    <arg name="vehicle" default="mono"/>


        <node
                pkg="mv_camera" 
                type="mv_camera_node" 
                name="mono_camera_node" 
                output="screen">
        <param name="guid" value="MV_CAMERA_25000813" />
        <param name="frame_rate" value="20" />
        <param name="pixel_clock" value="40."/>
        <param name="use_ros_time" value="true" />
        <remap from="camera" to="mono" />
        </node>


    <!-- configure the cameras -->
    <node   
            pkg="mv_camera" 
            name="cam_configurer"
            type="modify_mono_settings" 
            output="screen">
    <param name="ext_trig" value="false"/>
    <param name="binn_mode" value="Off"/>
    <param name="auto_expo" value="true"/>
    <param name="expo_mode" value="Standard"/>      
    <param name="expo_time" value="10000"/>
    <param name="auto_gain" value="On"/>
    <param name="hdr_mode" value="Off"/>
    </node>

    <!--Overlapped 
    BinningHV Off-->

</launch> 

